### PR TITLE
[PM-3810] Unify Passkeys view

### PR DESCRIPTION
**Pull request by gbubemismith**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Due to a change in the storage architecture for passkeys, the client UI needs to be unified to allow discoverable and non-discoverable keys.

## Screenshots
Screenshot 2023-09-18 at 4 58 21 PM
Screenshot 2023-09-18 at 4 58 40 PM
Screenshot 2023-09-18 at 4 58 57 PM
Screenshot 2023-09-18 at 5 09 48 PM
Screenshot 2023-09-18 at 5 21 48 PM
Screenshot 2023-09-18 at 5 22 09 PM

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/browser/src/vault/popup/components/vault/add-edit.component.html [hunk 1]
```diff
@@ -138,7 +138,8 @@ <h2 class="box-header">
             <div class="box-content">
               <div class="box-content-row text-muted">
                 <span class="row-label">{{ "typePasskey" | i18n }}</span>
-                {{ "passkeyTwoStepLogin" | i18n }}
```
Comment (coroiu) [inline apps/browser/src/vault/popup/components/vault/add-edit.component.html:141]: question: should this string be removed from the translation file?

File: apps/web/src/app/vault/individual-vault/add-edit.component.html [hunk 2]
```diff
@@ -201,7 +201,11 @@ <h1 class="modal-title" id="cipherAddEditTitle">{{ title }}</h1>
                     class="form-control"
                     type="text"
                     name="Login.Fido2key"
-                    [value]="'passkeyTwoStepLogin' | i18n"
+                    [value]="
+                      ('dateCreated' | i18n) +
+                      ' ' +
+                      (cipher.login.fido2Keys[0].creationDate | date : 'short')
+                    "
```
Comment (coroiu) [inline apps/web/src/app/vault/individual-vault/add-edit.component.html:208]: suggestion: maybe make this into a getter to simplify the html

File: libs/common/src/vault/models/view/cipher.view.ts [hunk 3]
```diff
@@ -36,7 +35,6 @@ export class CipherView implements View, InitializerMetadata {
   identity = new IdentityView();
   card = new CardView();
   secureNote = new SecureNoteView();
-  fido2Key = new Fido2KeyView();
   attachments: AttachmentView[] = null;
```
Comment (coroiu) [inline libs/common/src/vault/models/view/cipher.view.ts:38]: question: does this PR remove all reference to `CipherType.Fido2Key` ? The Jira ticket for this PR includes a task to remove it

> Remove the cipher type for discoverable passkeys and the accompanying UI

File: apps/browser/src/vault/popup/components/vault/add-edit.component.html (hunk 1 above)
Comment (gbubemismith) [reply apps/browser/src/vault/popup/components/vault/add-edit.component.html:141]: Yeah it should

File: libs/common/src/vault/models/view/cipher.view.ts (hunk 3 above)
Comment (gbubemismith) [reply libs/common/src/vault/models/view/cipher.view.ts:38]: Yeah this PR does

Comment (gbubemismith) [reply libs/common/src/vault/models/view/cipher.view.ts:38]: Oh I read that as the standalone field

**Review by coroiu — APPROVED** [review]
@gbubemismith I'm approving, but can we get someone else from Vault to look at these changes too before merging?

**Review by jlf0dev — APPROVED** [review]
LGTM

**Pull request merged**